tests/test_compressed_cartesian_mapping.cpp

```diff
@@ -26,25 +27,107 @@
 #include <opm/grid/utility/compressedToCartesian.hpp>
 #include <opm/grid/utility/cartesianToCompressed.hpp>
 
+#include <unordered_map>
+#include <iterator>
+
 BOOST_AUTO_TEST_CASE(mapping)
 {
     const std::vector<int> global_cell{0, 1, 2, 3, 5, 6, 8, 9};
+    const int num_cells = global_cell.size(); // 8
+
+    const std::unordered_multimap<int, int> cartesian_to_compressed =
+        Opm::cartesianToCompressed(num_cells, global_cell.data());
+
+    for (const auto pair : cartesian_to_compressed)
+    {
+        std::cout << pair.first << " " << pair.second << '\n';
+    }
```

Comment:
Is this code left over from development?  If so, I think we should remove it.

Reply:
Opps! No, that was supposed to be removed. Thanks for noticing!